packages/react-components/src/components/utils/ChatMessageComponentUtils.tsx

```diff
@@ -82,20 +88,18 @@ export function getMessageBubbleContent(
  */
 /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
 const defaultOnRenderFileDownloads = (
-  userId: string,
   message: ChatMessage | /* @conditional-compile-remove(data-loss-prevention) */ BlockedMessage,
   strings: MessageThreadStrings,
-  /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
-  fileDownloadHandler?: FileDownloadHandler
+  actionForAttachment?: (attachment: AttachmentMetadata, message?: ChatMessage) => AttachmentMenuAction[]
 ): JSX.Element | undefined => {
   /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
   return (
     <_AttachmentDownloadCards
-      userId={userId}
+      message={message as ChatMessage}
```

Comment:
Why do we need this cast?
How can we avoid this cast?

Reply:
the message we got here has 2 types" 
<img width="951" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/b72cd140-880b-4118-95db-3ddd00cbbe54">
we only need chat message for our attachment card, therefore we need a cast here

Reply:
if you get a type you're not expecting it will do what?

Reply:
it's an optional props, being undefined is ok